webui/module/Restore/src/Restore/Controller/RestoreController.php

```diff
@@ -238,7 +239,172 @@ public function indexAction()
 
    }
 
-   /**
+    public function versionsAction()
```

Comment:
Diese Funktion könnte in kleinere, besser verständliche Funktionen aufgeteilt werden.